### ISSUE TITLE
~/.authinfo never loaded

### ISSUE THREAD

**Issue by patrick-motard** [ahungry/org-jira#193]
Is ~/.authinfo ever actually loaded? I've filled in the file with the correct information but each time i call a function from org-jira it asks for my name and password. If i type in the name and password it works so i know i have the correct information. Looking through the source code, I don't see anywhere where it actually reads that file.

**Comment by utrack** [comment]
Same here, even after calling (netrc-parse) in ielm.

**Comment by ahungry** [comment]
It used to, so maybe something inadvertently got removed that used to refer to the file.

**Comment by patrick-motard** [comment]
Because of this bug, I have to manually log in each time, which is annoying especially considering I can't paste my password in the password prompt. Either that or I use a token which I don't particularly want to do (these tokens expire in the same way the browser session expires right?)

**Comment by ahungry** [comment]
the reference is in jiralib.el, the function responsible is `jiralib-login` - the snippet that does the reading is `auth-source-search` and it seems to work fine for me.

What version etc. of Emacs are you using?

**Comment by patrick-motard** [comment]
``` 
emacs --version                
GNU Emacs 26.2
Copyright (C) 2019 Free Software Foundation, Inc.
GNU Emacs comes with ABSOLUTELY NO WARRANTY.
You may redistribute copies of GNU Emacs
under the terms of the GNU General Public License.
For more information about these matters, see the file named COPYING.
```

**Comment by weipe** [comment]
It works for me - although I use ~/.authinfo.gpg.
```
$ emacs --version
GNU Emacs 26.3
Copyright (C) 2019 Free Software Foundation, Inc.
GNU Emacs comes with ABSOLUTELY NO WARRANTY.
You may redistribute copies of GNU Emacs
under the terms of the GNU General Public License.
For more information about these matters, see the file named COPYING.
```

**Comment by david-come-pix4d** [comment]
I had a similar issue until I removed the http:// from the machine

**Comment by ndunn990** [comment]
I know it's been a while since anyone commented on this, but it's still open and I encountered the issue, as well. As far as I can tell, everything was correct in my ~/.authinfo file. So, I went ahead and encrypted it for kicks (I was planning to do this, anyway) and it suddenly it prompted me to decrypt the file and started to work as expected

So:

- ~/.authinfo  >>  did not work
- ~/.authinfo.gpg  >>  _did_ work

Hopefully this helps someone.
